lib/test/readme.txt

```diff
@@ -0,0 +1,3 @@
+AspectJ_JDK_Update
+
+Always add latest aspectjrt.jar here to ensure that relevant integration tests run with the latest versions.
```

Comment:
Isn't that something the build tool can do automatically before each test run? It's probably a pita with Maven specifically, but might be worth the it. Alternatively a small test case that detects an out-of-date `aspectjrt.jar` might help too.

Reply:
You are right, of course. I am aware of that. I am simply in a tight spot concerning the time at my disposal. The actual problem is that this stuff was ever checked into the SCM at all. I eliminated lots of these relics already since I joined the project. These are a bit difficult, because the tests run before the main AspectJ build artifacts (runtime, matcher, weaver, tools) are being built, i.e. we have a circular dependency there. In the future, I will rather spend time fixing the root cause than automating the workaround.